WarOfEmpires/Views/Empire/Troops.cshtml

```diff
@@ -46,6 +46,24 @@
     @Html.HiddenFor(m => m.CavalryTrainingCost)
     @Html.HiddenFor(m => m.FootmanTrainingCost)
     @Html.HiddenFor(m => m.MercenaryTrainingCost)
+    @Html.HiddenFor(m => m.CurrentStamina)
+
+    <div class="row">
+        <div class="col-4 col-form-label">
+            @Html.LabelFor(m => m.Stamina)
+        </div>
+        <div class="col-2 col-form-label">
+            @Model.CurrentStamina.ToString(StringFormat.Integer)
+        </div>
+        <div class="col-2">
+            @Html.TextBoxFor(m => m.StaminaToFull, new { @class = "form-control" })
+            @Html.ValidationMessageFor(m => m.StaminaToFull, null, new { @class = "small form-text text-danger" })
+        </div>
+        @*TODO: Make the text selected, for quicker entry of the desired stamina to heal*@
```

Comment:
Can put this in GUI or something as a general thing

Reply:
Removed the comment and added a project in GUI